docs/guide/docs/infrastructure/performances.md

```diff
@@ -10,6 +10,45 @@ title: Performances
 
 ## Benchmark Tool
 
-Run Botpress with the `bench` command from the command line. You can run this on the Pro binary downloaded from the website. Make sure to adjust the `--url` and `--botId` to match your botpress server url and the ID of your bot.
-
+Run Botpress with the `bench` command from the command line. You can run this on the Pro binary downloaded from the website. Make sure to adjust the `--url` and `--botId` to match your botpress server url and your bot's ID. For more information on options, you can set for your test, run `./bp bench --help`
```

Comment:
Pro Binary? What do you mean by that? bench can only be run if you have a pro license?